projects/storefrontlib/src/layout/layout.module.ts

```diff
@@ -1,9 +1,13 @@
 import { NgModule } from '@angular/core';
+import { provideDefaultConfig } from '@spartacus/core';
 import { OutletRefModule } from '../cms-structure/outlet/outlet-ref/outlet-ref.module';
+import { defaultDirectionConfig } from './direction/default-direction.config';
+import { HtmlDirProvider } from './direction/direction.provider';
 import { LaunchDialogModule } from './launch-dialog/index';
 
 @NgModule({
   imports: [OutletRefModule, LaunchDialogModule.forRoot()],
   exports: [OutletRefModule],
+  providers: [HtmlDirProvider, provideDefaultConfig(defaultDirectionConfig)],
```

Comment:
Bagatelle: for separation of concerns, what do you think about encapsulating those providers in a new `DirectionModule` that could be imported here?

Reply:
Yes, I like that. I've moved direction related stuff to the Direction folder / module. The config is also no longer part of the layout config.